pkg/clients/exec_command.go

```diff
@@ -322,6 +327,9 @@ func (c *ContainerCreationExecContext) CreatePodAndWait() error {
 		}
 	}
 	if !running {
+		if c.noPodLifeCycle {
```

Comment:
What does it mean that the lifecycle is disabled?

Reply:
It wont try to create/delete the debug pod, I don't like the name but couldn't think of a better one.

Reply:
I changed it to `unmanagedDebugPod` which I think matches intent better.